docs/source/algorithms/IndirectILLReductionDIFF-v1.rst

```diff
@@ -0,0 +1,45 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm is in charge of reducing data acquired using **IN16B** indirect geometry instrument at **ILL**, when used in diffractometer mode.
```

Comment:
This algorithm reduces data acquired by the diffraction detectors of **IN16B** indirect geometry instrument at **ILL**.